src/validations/index.js

```diff
@@ -1 +1,60 @@
-// this folder will contain validation middlewares' definitions.
\ No newline at end of file
+// this folder will contain validation middlewares' definitions.
+import Joi from 'joi';
+
+const schema = {
+	signupvalidate: Joi.object().keys({
+		email: Joi.string().required().email(),
+		firstName: Joi.string()
+			.empty()
+			.min(2)
+			.max(20)
+			.pattern(/^[a-zA-Z]/)
+			.messages({
+				'any.required': '{{#label}} field is required',
+				'string.base': '{{#label}} must be of type string',
+				'string.empty': '{{#label}} can not be empty',
+				'string.pattern.base':
+					'{{#label}} must contain only characters from a to z.',
```

Comment:
you are validating first name and last name but you have name in model can you make it much?

Reply:
i don't see the problem in my codes there is no name